server/db/models/index.js

```diff
@@ -34,7 +34,7 @@ Jelly.hasMany(Review)
 Review.belongsTo(User)
 User.hasMany(Review)
 Category.hasMany(Jelly)
-Jelly.belongsTo(Category)
+Jelly.belongsToMany(Category)
```

Comment:
breaking change here. it's making tests fail because it needs a through association for the table